src/osp/Universe.h

```diff
@@ -71,7 +70,12 @@ using MapSatType = std::map<std::string, std::unique_ptr<ITypeSatellite>>;
  */
 class Universe
 {
+    using MapTypeSats_t = std::map<std::string_view, TypeSatIndex, std::less<>>;
```

Comment:
The string_view is pointing to a location that's stable in memory, right?

Reply:
Yes, all of these point to constants